src/allmydata/web/root.py

```diff
@@ -28,7 +28,7 @@
     WebError,
     get_arg,
     MultiFormatPage,
-    RenderMixin,
+    MultiFormatResource,
```

Comment:
:+1: great .. when the last thing stops using `RenderMixin` we can delete it too :)

Reply:
Four more uses of `RenderMixin` now.  I guess it can be done!